build.gradle

```diff
@@ -277,7 +277,7 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.3-prerelease'
```

Comment:
This needs to be updated to a full release version of the hmcts/ccd-test-defintions before going into master. see hmcts/ccd-test-definitions#51.

Reply:
Resolved in f31e8b632686248b0e1f8844f8fd8c34a4719ac7.

Reply:
Done in [f31e8b6](https://github.com/hmcts/ccd-data-store-api/pull/2058/commits/f31e8b632686248b0e1f8844f8fd8c34a4719ac7)